### PR TITLE
docs: add link to wiki page

### PR DESCRIPTION
**Pull request by nfischer**
Pretty trivial change, just wanted to see what @ariporad thinks of it. It gives a shoutout to the two most popular projects in the wiki.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -23,7 +23,10 @@ The project is [unit-tested](http://travis-ci.org/shelljs/shelljs) and battled-t
 
 If you have feedback, suggestions, or need help, feel free to post in our [issue tracker](https://github.com/shelljs/shelljs/issues).
 
-For a non-programmatic implementation of Unix shell commands on top of Node.js, check out [Cash](https://github.com/dthree/cash).
+Think ShellJS is cool? Check out some related projects (like
+[cash](https://github.com/dthree/cash)--a javascript-based POSIX shell--or
+[shx](https://github.com/shelljs/shx)--ShellJS functions exposed straight to
```
Comment (ariporad) [inline README.md]: Two things:
1. I think the wiki page should be a separate item.
2. I feel like `shx` should get it's own section. Maybe with a little intro.

Comment (nfischer) [inline README.md]: @ariporad could you rewrite this PR to clarify what you mean?

Comment (ariporad) [inline README.md]: Sure.

Comment (ariporad) [inline README.md]: @nfischer: Done. (I made it a separate commit so you could compare the changes, I'll squash them if you like them).

**Comment by nfischer** [comment]
@ariporad I rebased this. I changed the wording a little bit. I also removed the docs for the `make` command, and moved these to the wiki (and a link is provided). The rationale behind this is that I want less junk in the way of the command reference, which is probably the most important part of the README.

**Comment by ariporad** [comment]
LGTM!

**Pull request merged**